lua/vsrocq/client.lua

```diff
@@ -188,29 +188,29 @@ commands[#commands + 1] = 'admitted'
 commands[#commands + 1] = 'goals'
 
 -- TODO: commands in panels
-function VSCoqNvim:ensure_proofview_panel()
+function VSRocqNvim:ensure_proofview_panel()
   if vim.api.nvim_buf_is_valid(self.proofview_panel) then
     if not vim.api.nvim_buf_is_loaded(self.proofview_panel) then
       vim.fn.bufload(self.proofview_panel)
     end
     return
   end
   self.proofview_panel = vim.api.nvim_create_buf(false, true)
-  vim.bo[self.proofview_panel].filetype = 'coq-goals'
+  vim.bo[self.proofview_panel].filetype = 'rocq-goals'
```

Comment:
we're using coqtail's coq-goals and coq-infos syntax, so we should keep this unchanged